### PR TITLE
Cura 3.1 Uranium - Polish language

### PR DESCRIPTION
**Pull request by anraf1001**
Translated uranium.po file for Cura 3.1

### CODE REVIEW

File: resources/i18n/pl_PL/uranium.po [hunk 1]
```diff
@@ -356,7 +356,7 @@ msgstr "Nie znaleziono ważnych rozszerzeń OpenGL. Ten program wymaga wsparcia
 #, python-brace-format
 msgctxt "@info:status Don't translate the XML tag <filename>!"
 msgid "Cannot open files of the type of <filename>{0}</filename>"
-msgstr ""
+msgstr "No można otworzyć plików typu <filename>{0}</filename>"
```
Comment (thopiekar) [inline resources/i18n/pl_PL/uranium.po]: Isn't it "Nie można.."?

**Comment by thopiekar** [comment]
@anraf1001 I looked over your PR.
I'm only not sure about one thing, but the rest seems to be fine.
However, polish is not my native language, so you might be right.

Dziękuję za twoja praca!

File: resources/i18n/pl_PL/uranium.po (hunk 1 above)
Comment (anraf1001) [reply resources/i18n/pl_PL/uranium.po]: Oh. Sorry. That's my mistake. Thanks for being patient.

Comment (thopiekar) [reply resources/i18n/pl_PL/uranium.po]: No problem. Looks like my Polish is still not too bad 😋

**Comment by thopiekar** [comment]
@Ghostkeeper Did you take a look on this?

**Comment by anraf1001** [comment]
@thopiekar as I know. Ghostkeeper knows about my every translation :-)

**Comment by Ghostkeeper** [comment]
I know about it, but we haven't taken the time yet to review it. Probably on Thursday or Friday. I'll sit together with QA to go through most of the translations at once.

To update all of our translators at once: We expect to get the translations from Bothof (our usual translation bureau) on Thursday for German, Spanish, French, Italian, Dutch, Russian, Turkish, Japanese, Korean and simplified Chinese. Russian used to be done by @RaD, but Ultimaker decided to get Bothof to take it over since RaD wanted to stop. Finnish will be dropped this release (used to be done by Bothof). Brazillian Portuguese is being done by @Patola but I don't know how far he is. We'll probably also get a Portuguese Portuguese translation soon by Bothof as well, but not in time for our release so I don't know yet if that'll be for 3.1 or 3.2.

**Comment by Ghostkeeper** [comment]
Developers, this falls under CURA-4601 (the ticket about translations of 3.1). See also https://github.com/Ultimaker/Cura/pull/2826

**Pull request merged**